src/labels/CustomLabels.labels

```diff
@@ -4883,6 +4883,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>healthDetailsInvalidRDCustomPeriodPIcklist</shortDescription>
         <value>Recurring Donation Installment Period picklist value {0} is not a valid Custom Installment Period.</value>
     </labels>
+    <labels>
```

Comment:
@sam-knox Would you mind reviewing these new custom labels? These are used to display recurring donation health check errors for either unsupported picklist values or missing required picklist values. There are existing labels for some of the other recurring donations health check operations that are unchanged. Thanks.

Reply:
@force2b - Will it be clear to the user what field the error is on? Just saying that a picklist value isn't supported might lead to the question: Which picklist?

Reply:
Good question. The related _solution_ label will have both the picklist value (`{0}`) and the label of the picklist field (`{1}`) to fix.